### DISCUSSION TITLE
Merging the features of c1, country1

### DISCUSSION THREAD

**Discussion by amerfogbugz** [mohamedhamdyamer/ggt#13] [Ideas]
hello @mohamedhamdyamer ,

how about creating a new pull request to merge what we have done for c1, country1, into master?

i think the features were great and can be part of our core app.

let me know,

**Comment by mohamedhamdyamer** [comment]
sounds great, @amerfogbugz !

let's do ;-)

I will create an issue and will assign it to you

**Comment by amerfogbugz** [comment]
thanks!

we've done a great job with such a merge ;-)

**Comment by mohamedhamdyamer** [comment]
thanks fogbugz,

appreciate you help

**Comment by mohamedhamdyamer** [comment]
please close the discussion!

**Comment by amerfogbugz** [comment]
better not to close the discussion from my side

**Comment by amerfogbugz** [comment]
@mohamedhamdyamer ,

Could you do it from your side?

**Comment by mohamedhamdyamer** [comment]
sure,

closing ...

**Comment by amerfogbugz** [comment]
thanks :-)
